src/main/java/org/terasology/thirst/ThirstAuthoritySystem.java

```diff
@@ -71,6 +71,7 @@
     private DelayManager delayManager;
 
     private boolean destroyDrink;
+    private float expectedDecay;
```

Comment:
I don't see why this should be a class member - it is only used in `characterMoved` method.

```suggestion
```